### DISCUSSION TITLE
Question about the release tag prefix `v`.

### DISCUSSION THREAD

**Discussion by razonyang** [lucide-icons/lucide#1622] [Q&A] [answered]
Hi, I noticed that the release tag removes the leading `v`, is it intentional?

**Comment by ericfennis** [comment, marked as answer]
Uhm good question. No and Yes. It was by mistake, but we stick with it for now.
But maybe we will revert it. Not sure yet.

**Reply by razonyang** [reply]
I see, I've one project (Go module) that relies the leading `v` for versioning, but it's not a big deal, I can fork and re-tag the release.
Will keep watching for subsequent releases, thank you so much for your explaination.
